### DISCUSSION TITLE
Log Value

### DISCUSSION THREAD

**Discussion by GimmeMe1** [jomjol/AI-on-the-edge-device#969] [Q&A]
is it possible to log recognized values in a file for the last 5 days, format: date-time-value or similar?

**Comment by jomjol** [comment]
Technically no problem. But can filter them with a little effort also from the normal log file.

If you implement an additional file into the code, just make a pull request and I will check and merge it.

**Reply by GimmeMe1** [reply]
yup you are right, the values are logged in the main log, so it should be possible somehow to get it from there as well. But actually i would prefere a separate log. Since I can't code I will try to go with this way. If there are more people asking for the feature I would be happe if someone can code it. Best
